docs/UserGuide.md

```diff
@@ -148,6 +148,9 @@ amount, category and remarks of a given expense, using the delimiters of `d/` ,
 respectively.
 - The DATE field must be in the format of YYYY-MM-DD. All other formats would not be accepted.
 - The AMOUNT field must be a valid number (integer/float) and must be non-negative.
+- '/' must not be the last character of a given field, even with trailing spaces.
```

Comment:
I trimmed trailing spaces away while also checking that '/' characters are surrounded by whitespaces. If the '/' is in the last character, it will always fail since the spaces after it are trimmed, so this constraint is just to prevent any slashes at the end. I don't think it makes sense from a user perspective to have the last character as a slash, but I'm happy to hear alternate suggestions!